### PR TITLE
chore(ci): add `--debug --verbose` to stencil nightly build

### PR DESCRIPTION
**Pull request by alicewriteswrongs**
Set the `--debug` and `--verbose` flags on the Stencil Nightly CI build.

## What is the current behavior?

The Stencil nightly build doesn't provide all the information that it could!

## What is the new behavior?

This sets the `--debug` and `--verbose` flags when calling `npm run build` in the 'Build Ionic Core with Stencil Prerelease' github action, which is used by the workflow which installs nightly Stencil builds and builds framework with it, thereby checking for regressions.

This change will just ensure that this nightly build provides a bit more information.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by mapsandapps — APPROVED** [review]
good to merge once we get the builds fixed 😬

**Comment by rwaskiewicz** [comment]
Created https://github.com/ionic-team/ionic-framework/pull/29079 to fix CI, if it doesn't magically fix itself in the mean time

**Comment by rwaskiewicz** [comment]
@alicewriteswrongs should be good to go after a rebase, #29079 landed

**Pull request merged**
